tutorials/monitoring/changedetection-1-basics/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Changedetection Basics
+description: Learn the basic features of Changedetection, including monitoring websites and receiving notifications.
+layout: page
+parent: Monitoring
+grand_parent: Tutorials
+---
+
+# How to Monitor Website Changes with Changedetection?
+
+Changedetection is an open-source app that lets you monitor website changes. You can automate the entire process. From simply monitoring website pages that have a change (such as watching prices, restock notification), to deep inspection such as PDF text support, JSON and XML monitoring and extensive text triggers. Here is our step-by-step guide on how you can set up website monitoring.
+
+# How to add and monitor a website with Changedetection?
+
+<video controls loop muted preload="none" src="1-add-monitor-website.mp4" poster="1-add-monitor-website.webp">
+</video>
+
+You can monitor any website for changes with Changedetection. You can track changes for the entire website, or track only one page. You can also monitor for out of stock items. Here are the steps to adding website that you want to track:
+
+1. **Copy the URL** that you want to monitor for changes.
+2. **Paste the URL** in the empty field.
+3. **Add the tag** or label.
+4. Click on '**Watch**'.
+5. Click on '**Recheck**' to force the website check.
+6. Click on '**Diff**' to see website changes.
+
+# How to edit tracking settings?
```

Comment:
... edit monitoring settings

(tracking isn't the right word here. It's a monitoring app)